backpack/core/derivatives/batchnorm_nd.py

```diff
@@ -323,16 +320,3 @@ def _get_free_axes(
         for n in range(self._get_n_axis(module)):
             free_axes.append(index_batch + n + 2)
         return tuple(free_axes)
-
-    @staticmethod
-    def _maybe_warn_no_batch_summation(sum_batch: bool) -> None:
```

Comment:
What's the reason to remove this?

Reply:
See [here](https://github.com/f-dangel/backpack/pull/202#discussion_r680965171).
Resolve.